### ISSUE TITLE
"scenesAsSwitches" doesn't work

### ISSUE THREAD

**Issue by derthobo** [ebaauw/homebridge-hue#890]
#### Issue
I'm using the "scenesAsSwitches"-option with deconz (stable). Since version 0.12.14 the switches doesn't work anymore. After a downgrade to version 0.12.13 everything works fine.

#### Log Messages

```
[1/12/2021, 10:05:03 PM] [Hue] deconz: recall scene "Wohnzimmer HELL"
(node:554) UnhandledPromiseRejectionWarning: TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at numberOfZigbeeMessages (/homebridge/node_modules/homebridge-hue/lib/HueClient.js:24:14)
    at HueClient.put (/homebridge/node_modules/homebridge-hue/lib/HueClient.js:345:21)
    at HueBridge.put (/homebridge/node_modules/homebridge-hue/lib/HueBridge.js:1139:27)
    at On.<anonymous> (/homebridge/node_modules/homebridge-hue/lib/HueAccessory.js:183:27)
    at On.emit (events.js:315:20)
    at On.EventEmitter.emit (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/EventEmitter.ts:42:22)
    at On.Characteristic._this.setValue (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:652:12)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1461:24
    at Array.forEach (<anonymous>)
    at Bridge.Accessory._this._handleSetCharacteristics (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1303:10)
    at HAPServer.emit (events.js:315:20)
    at HAPServer.EventEmitter.emit (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/EventEmitter.ts:42:22)
    at HAPServer._this._handleCharacteristics (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/HAPServer.ts:812:12)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/HAPServer.ts:280:24)
    at IncomingMessage.emit (events.js:327:22)
(node:554) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
```

**Comment by ebaauw** [comment]
Bloody hell, the deCONZ API uses a PUT without a body to recall a scene.

**Comment by ab1781** [comment]
I habe noticed the very same bug, destroying my setup based on scenes a lot. Have to go back to previous version (glad to have a backup ;-)). Thanks a lot in advance for a fix :-)

**Comment by ebaauw** [comment]
Fixed in v0.12.15.
